ui/admin/app/components/form/role/manage-scopes/index.hbs

```diff
@@ -0,0 +1,88 @@
+<Rose::Form
+  class='full-width role-manage-scopes-form'
+  @onSubmit={{fn this.submit @submit}}
```

Comment:
Why do you need `this.submit`? It looks like it just passes through anyway?

Does this not work?
```suggestion
  @onSubmit={{fn @submit this.selectedItems}}
```

Reply:
that should work. originally had it the other way thinking i was going to add more logic but that isn't necessary yet. good catch!